src/main/java/net/mcreator/workspace/ShareableZIPManager.java

```diff
@@ -102,18 +102,22 @@ public static File importZIP(File file, File workspaceDir, Window window) {
 	}
 
 	public static void exportZIP(String title, File file, MCreator mcreator, boolean excludeRunDir) {
-		ProgressDialog dial = new ProgressDialog(mcreator, title);
+		exportZIP(title, file, mcreator.getWorkspaceFolder(), mcreator, excludeRunDir);
+	}
+
+	public static void exportZIP(String title, File file, File workspaceFolder, Window parent, boolean excludeRunDir) {

```

Comment:
Why is this overload needed? MCreator can already make backups without this overloaded method.

Reply:
I didn't see I could change a small thing and avoid this.